apps/site/assets/css/_amenity.scss

```diff
@@ -15,3 +15,47 @@
     margin: .65rem;
   }
 }
+
+.access-amenities-header {
+  background: $brand-primary-darkest;
+  color: $white;
+  height: 3.75rem;
+}
+
+tr:nth-child(2n+3) {
+  background-color: $gray-lightest;
+}
+
+.access-amenities-row {
+  height: 5rem;
+}
+
+.status {
+  padding-right: 3.125rem;
+}
+
+.access-amenities-name {
+  padding-left: 1rem;
+}
+
+.access-amenities-text {
+  font-size: .875rem;
+  font-weight: 400;
+}
+
+.amenity-status {
+  padding-right: .625rem;
+}
+
+.amenity-working {
+  color: green;
+}
+
+.amenity-out {
+  color: red;
```

Comment:
I think both of these should be color variables that we have defined in our `_colors.scss` file to help keep things consistent.  Perhaps the `$error-text` color for red and `$brand-success` for green